### PR TITLE
feat(preset-mini): support content-raw-[xxx]

### PR DESCRIPTION
**Pull request by sudongyuer**

### CODE REVIEW

File: packages/preset-mini/src/_rules/static.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ export const contentVisibility: Rule[] = [
 
 export const contents: Rule[] = [
   [/^content-\[(.+)\]$/, ([, v]) => ({ content: `"${v}"` })],
+  [/^content-raw-\[(.+)\]$/, ([, v]) => ({ content: `"${v}"` })],
```
Comment (xhebox) [inline packages/preset-mini/src/_rules/static.ts:73]: Should be "content: `${v}`"

Comment (antfu) [inline packages/preset-mini/src/_rules/static.ts:73]:
```suggestion
  [/^content-raw-\[(.+)\]$/, ([, v]) => ({ content: v })],
```

Isn't this supported to be like this?

**Comment by chu121su12** [comment]
Can I suggest alternate syntax of `content-[raw:x]` or `content-[string:x]` instead?

**Comment by chris-zhu** [comment]
> Can I suggest alternate syntax of `content-[raw:x]` or `content-[string:x]` instead?
> 
> Edit: using `h.bracket` and something [like this](https://github.com/unocss/unocss/commit/43f0b48ec011df6b36f192c2412fff337ae65bc7)

Cool, I prefer this syntax.

**Pull request closed without merging**